### PR TITLE
add security considerations back to User Guide

### PR DESCRIPTION
**Pull request by rosemarymarano**
This text came from the v1.1 README. Monica wanted it put back into the User Guide.

### CODE REVIEW

File: site/user-guide.md [hunk 1]
```diff
@@ -66,6 +66,21 @@ You can have one or more operators in your Kubernetes cluster that manage one or
...
 
+## Exposing applications outside the Kubernetes cluster
+The operator can configure services to expose WebLogic applications and features outside of the Kubernetes cluster.  Care should be taken when exposing anything externally to ensure that the appropriate security considerations are taken into account. In this regard, there is no significant difference between a WebLogic domain running in a Kubernetes cluster and a domain running in a traditional data center.  The same kinds of considerations should be taken into account, for example:
+
+* Only expose those protocols and ports that need to be exposed.
+*	Use secure protocols (HTTPS, T3S, and such).
+*	Use custom channels to restrict the protocols that are exposed.
+*	Is load balancing required?
+*	Is certificate-based integrity needed?
+*	How will users authenticate?
```
Comment (tbarnes-us) [inline site/user-guide.md]: Add:
Is exposing RMI required (T3 enables RMI), or IIOP, or would HTTP alone suffice?
Is it necessary to expose admin capable channels?

Comment (tbarnes-us) [reply site/user-guide.md]: Is it required to expose default channels (these accept T3/RMI, and by default, accept admin traffic)?

Comment (rosemarymarano) [reply site/user-guide.md]: I have added your suggestions.

File: site/user-guide.md [hunk 2]
```diff
@@ -66,6 +66,24 @@ You can have one or more operators in your Kubernetes cluster that manage one or
...
+* Is the network channel encrypted?
+* Is exposing RMI required (T3 enables RMI), or IIOP, or would HTTP alone suffice?
+* Is it necessary to expose admin-capable channels?
+* Is it required to expose default channels (these accept T3/RMI, and by default, accept administrative traffic)?
+
+While it is natural to expose web applications outside the cluster, exposing administrative features like the Administration Console and a T3 channel for WLST should be given more careful consideration.  There are alternative options that should be weighed.  For example, Kubernetes provides the ability to securely access a shell running in a container in a pod in the cluster.  WLST could be executed from such an environment, meaning the T3 communications are entirely within the Kubernetes cluster and therefore more secure.
+
+Oracle recommends careful consideration before deciding to expose any administrative interfaces externally.
```
Comment (tbarnes-us) [inline site/user-guide.md]: any administrative interfaces --> any administrative or non-HTTP(S) interfaces or protocols

**Pull request merged**